addons/mail/static/src/discuss/call/common/call_menu.js

```diff
@@ -1,14 +1,43 @@
-import { Component, useState } from "@odoo/owl";
+import { Component, useEffect, useState } from "@odoo/owl";
 
 import { registry } from "@web/core/registry";
 import { useService } from "@web/core/utils/hooks";
+import { callActionsRegistry } from "./call_actions";
 
 export class CallMenu extends Component {
     static props = [];
     static template = "discuss.CallMenu";
     setup() {
         super.setup();
         this.rtc = useState(useService("discuss.rtc"));
+        this.state = useState({
+            icon: undefined,
+        });
```

Comment:
Nitpicking: you be a single line for defining `this.state = useState({ icon: undefined });`. Feel free to help prettier when you can. Again balancing content on X and Y axis helps having macro-view of file content at a glance.